docs/ensembl-help/using-ensembl/ensembl-apps/blast/blast.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: blast
+title: How to run BLAST
+description: How to run BLAST queries
+related_articles:
+tags:
+  - blast
+status: draft
+---
+
+# How to run BLAST
+
+BLAST is a sequence similarity search tool that can be used for both DNA and proteins. 
+
+## Selecting BLAST parameters
+
+The target databases available for similarity searches are DNA, transcripts and proteins. Click on the ‘Database’ dropdown menu and choose from these options. 
+
+The following BLAST programs are available:
+- BLASTN: nucleotide sequences against nucleotide databases 
+- TBLASTX: translated nucleotide sequences against a translated nucleotide database
+
+The relevant BLAST program is selected automatically when the target database is selected and the Nucleotide or Protein option is selected above the sequence text box. The BLAST programs have pre-configured parameters, which you can view and change by clicking on the ‘Parameters’ option.
+
+## Entering a query sequence
+
+Paste the query sequence into the text box (the suggested format is FASTA) or upload a sequence file.
+ 
+Up to 50 sequences can be added by clicking on ‘Add another sequence’.
+
+![The BLAST submission page](media/BLAST.png)
+
+## Select target species
+
+You can select the species you want to run your sequence against by clicking on ‘Select species’ on the right hand side.
+
+## Submitting your BLAST query
+
+You can give a name or description to this BLAST query in the ‘Submission name’ (optional) field. Once your parameters are set, click RUN to start the search.
+
+## What does the result table show
+
+The submissions page will show the jobs that are currently running or recently completed. You have the option to Edit/rerun the job. Results are available for 7 days and queries can be rerun for 28 days.
+
+A submission ID is assigned to each job and additional information is provided, including the date and time of submission. The progression of the job gets automatically refreshed every 10 seconds until the job is fully completed. If you navigate away from the BLAST interface, the status of the query is indicated by the BLAST icon in the top panel changing from red to green to prompt you that it has successfully completed.
```

Comment:
Throughout this document, it may be useful to stick to the following terminology: what the user submits in a BLAST form, is a BLAST **submission**; which can consist of one or more **jobs**.  The reason this distinction may be useful is because any one of the jobs in submission may fail, whereas a submission as a whole may succeed. E.g. a submission may have three jobs, one of which fails and two succeed.

> The progression of the job gets automatically refreshed every 10 seconds until the job is fully completed.

10 seconds is a technical detail, almost certainly unnecessary to the user (), and also likely an incorrect one (I think we are checking job statuses every 15 seconds; but this periodicity may change). What would be useful to explain to the user though is that while any of the jobs in a submission is still running, they can not view the results of a submission.

Also, it may be worth explaining that while any of the jobs in one or more BLAST submissions are still running, we will show a red dot over the BLAST icon in the launchbar. If all jobs have finished running and there are some unviewed BLAST submissions, the red dot will change its colour to green. It disappears if all submissions in the unviewed list have been viewed.

Reply:
Thanks @azangru - I've updated this section of the article to be clearer about 'submissions' and 'jobs' and remove the mention of the page refresh frequency.